In file included from ../src/intel/vulkan/anv_queue.c:28:
In file included from ../src/intel/vulkan/anv_private.h:79:
In file included from ../src/vulkan/runtime/vk_command_buffer.h:29:
In file included from ../src/vulkan/runtime/vk_log.h:24:
In file included from ../src/vulkan/runtime/vk_instance.h:35:
../include/renderdoc_app.h:43:2: error: "Unknown platform"
   43 | #error "Unknown platform"
      |  ^

--- include/renderdoc_app.h.orig	2025-02-28 20:35:13 UTC
+++ include/renderdoc_app.h
@@ -35,12 +35,8 @@
 
 #if defined(WIN32)
 #define RENDERDOC_CC __cdecl
-#elif defined(__linux__)
-#define RENDERDOC_CC
-#elif defined(__APPLE__)
+#else
 #define RENDERDOC_CC
-#else
-#error "Unknown platform"
 #endif
 
 #ifdef __cplusplus
